plugins/carbon_black_cloud/unit_test/test_monitor_alerts.py

```diff
@@ -66,14 +67,14 @@ def setUp(self) -> None:
                 "first run",
                 task_first_run.copy(),
                 ("observation_id", "first_alerts", "first_observations"),
-                task_first_run_output,
+                task_first_run_output_with_offset,
```

Comment:
Chatted about this offline and I can see why the state has changed but I think this will be confusing for the next person to look at this. Also looking at the mocked responses I would expect `has_more_pages` to be True as we now contain the offset. 

could we update the times of the data in `first_observations` to be within the timeframe meaning no offset is applied. And then add a new unit test for this logic of adjusting the offset?